Team54/src/main/java/ISA/Team54/Examination/service/implementation/ExaminationServiceImpl.java

```diff
@@ -42,6 +47,11 @@
 public class ExaminationServiceImpl implements ExaminationService {
 	final long ONE_MINUTE_IN_MILLIS = 60000;//millisecs
```

Comment:
Ovo isto, cemu sluzi?

Reply:
Koristila sam to za konvertovanje vremena u nekoj od metoda tog servisa